packages/components/cascader/src/cascader.vue

```diff
@@ -24,7 +24,9 @@
   >
     <template #default>
       <div
-        v-clickoutside:[cascaderPanelRef]="() => togglePopperVisible(false)"
+        v-clickoutside:[cascaderPanelRef?.$el]="
+          () => togglePopperVisible(false)
```

Comment:
```ts
const contentRef = computed(() => {
      return tooltipRef.value?.popperRef?.contentRef
    })
```

Perhaps it would be more appropriate to restore the original method and change the name.

The `tooltipRef.value?.popperRef?.contentRef` contains both `cascaderPanelRef` and `suggestionPanel`

Reply:
Okay, but the verification side will be judged as unused, so I need to call him in expose to pass eslint.

Reply:
done.